src/utilities/helperFunctions.js

```diff
@@ -78,3 +80,22 @@ export const getCurrentDate = () => {
   };
   return currentDate.toLocaleDateString('en-US', options);
 };
+
+export const getHourAndMinString = (date, timezone) => {
+  const currentTime = new Date(date * 1000);
+  const currentMinute = currentTime.getMinutes().toString().padStart(2, '0');
+  let currentHour = currentTime.getHours();
```

Comment:
I think this should work for all timezones:

```suggestion
export const getHourAndMinString = (timestamp, timezone) => {
  const currentTimeUnadjusted = new Date((timestamp + timezone) * 1000);
  const currentMinute = currentTimeUnadjusted.getUTCMinutes().toString().padStart(2, '0');
  let currentHour = currentTimeUnadjusted.getUTCHours();
```